### ISSUE TITLE
ng add store --module wrong path

### ISSUE THREAD

**Issue by martzcodes** [ngrx/platform#1143]
## Minimal reproduction of the bug/regression with instructions:

Instructions:

```
ng new temp
cd temp
ng add @ngrx/store --module=app.module.ts
```

Output:

```
ng add @ngrx/store --module=app.module.ts
Installing packages for tooling via npm.
+ @ngrx/store@6.0.1
added 1 package from 1 contributor in 4.326s
[!] 13 vulnerabilities found [21863 packages audited]
    Severity: 9 Low | 4 High
    Run `npm audit` for more detail

Installed packages for tooling via npm.
Specified module does not exist
```

## Expected behavior:

It should install the module in `src/app/app.module.ts`.

Workaround:  using `ng add @ngrx/store --module=../app.module.ts` works

Doing the same thing with `ng add @ngrx/effects --module=app.module.ts` works fine.  I've gone through the schematics code comparing store with effects and don't see anything... 

## Versions of NgRx, Angular, Node, affected browser(s) and operating system(s):

```
ng --version

     _                      _                 ____ _     ___
    / \   _ __   __ _ _   _| | __ _ _ __     / ___| |   |_ _|
   / △ \ | '_ \ / _` | | | | |/ _` | '__|   | |   | |    | |
  / ___ \| | | | (_| | |_| | | (_| | |      | |___| |___ | |
 /_/   \_\_| |_|\__, |\__,_|_|\__,_|_|       \____|_____|___|
                |___/


Angular CLI: 6.0.8
Node: 10.1.0
OS: darwin x64
Angular: 6.0.6
... animations, common, compiler, compiler-cli, core, forms
... http, language-service, platform-browser
... platform-browser-dynamic, router

Package                           Version
-----------------------------------------------------------
@angular-devkit/architect         0.6.8
@angular-devkit/build-angular     0.6.8
@angular-devkit/build-optimizer   0.6.8
@angular-devkit/core              0.6.8
@angular-devkit/schematics        0.6.8
@angular/cli                      6.0.8
@ngtools/webpack                  6.0.8
@schematics/angular               0.6.8
@schematics/update                0.6.8
rxjs                              6.2.1
typescript                        2.7.2
webpack                           4.8.3
```

## I would be willing to submit a PR to fix this issue

[x] Yes (Assistance is provided if you need help submitting a pull request)
[ ] No

**Comment by martzcodes** [comment]
I overlooked a difference between the store and effects schematics files.

Effects does this:

```
    if (options.module) {
      options.module = findModuleFromOptions(host, options);
    }
```

While Store does this:

```
    if (options.module) {
      options.module = findModuleFromOptions(host, {
        name: '',
        module: options.module,
        path: options.path,
      });
    }
```

Store also redefines options.path prior to findModuleFromOptions (while effects does it after).

I should be able to track down the culprit and submit a PR

**Comment by martzcodes** [comment]
Was fixed in 9d360162397cc01de0ff867d45cc376e00575d87
